src/Mailer/Email.php

```diff
@@ -1507,7 +1507,6 @@ public static function deliver(
         if (is_array($config) && !isset($config['transport'])) {
             $config['transport'] = 'default';
         }
-        /* @var \Cake\Mailer\Email $instance */
```

Comment:
Are you sure you should remove this?
The discovery of setTo() method can not work without it.